.github/workflows/continuous.integration.yml

```
@@ -0,0 +1,56 @@
+name: Run Checks on PR
+
+on:
+  pull_request:
+    branches:
+      - main
+
+permissions:
+  actions: write
+  checks: write
+  contents: read
+  security-events: write
+  pull-requests: write
+
+env:
+  CI: true
+  SITE_DIR: site
+  TETRIS_APP_HOST: "127.0.0.1"
+  TETRIS_APP_PORT: "8080"
+  TETRIS_APP_PATH: "github-devsecops-fundamentals"
+
+jobs:
+  quality-checks:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-python@v4
+        with:
+          python-version: 3.12
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 20
+
+      - name: Install Dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements.ci.txt
+          npm ci
+          # Install browsers for running functional tests
+          npx playwright install --with-deps chromium
```

```suggestion
        run: |
          python -m pip install --upgrade pip
          pip install -r requirements.ci.txt
          npm ci
          npx playwright install --with-deps chromium
```

Remove pointless comment.